.gitignore

```diff
@@ -23,3 +23,10 @@ dist
 *.sln
 *.sw?
 .env
+node_modules/
+/test-results/
+/e2e/playwright-report/
```

Comment:
@IanMayo  - if we ignore `/e2e` the child folders will be ignore too, won't they?

Reply:
yes, if we just ignore `/e2e` the it'll ignore all child folders. In this case we are just ignoring `playwright-report` inside `e2e`

Reply:
Duh, of course :-)